### PR TITLE
KEP-2579: PodSecurity beta updates

### PR DESCRIPTION
**Pull request by liggitt**
Update PodSecurity KEP for beta

- Issue link: #2579 

/sig auth
/cc @tallclair @enj

### CODE REVIEW

File: keps/sig-auth/2579-psp-replacement/README.md [hunk 1]
```diff
@@ -258,10 +258,9 @@ leveraging the library implementation.
 
 When an `enforce` policy (or version) label is added or changed, the admission plugin will test each pod
 in the namespace against the new policy. Violations are returned to the user as warnings. These
-checks have a timeout of XX seconds and a limit of YY pods, and will return a warning in the event
+checks have a timeout of 1 second and a limit of 3,000 pods, and will return a warning in the event
 that not every pod was checked. User exemptions are ignored by these checks, but runtime class
-exemptions still apply. Namespace exemptions are also ignored, but an additional warning will be
-returned when updating the policy on an exempt namespace. These checks only consider actual Pod
+exemptions and namespace exemptions still apply. These checks only consider actual Pod
```
Comment (tallclair) [inline keps/sig-auth/2579-psp-replacement/README.md:263]: Maybe clarify that namespace exemptions are honored for dry-running pods, but exempt namespaces must still have valid labels?

Comment (liggitt) [reply keps/sig-auth/2579-psp-replacement/README.md:263]: scoped the exemptions statement to whether or not existing pods are evaluated

**Pull request merged**